packages/flutter/test/foundation/consolidate_response_test.dart

```diff
@@ -127,9 +127,9 @@ void main() {
     });
 
     group('when gzipped', () {
-      final Uint8List gzipped = gzip.encode(chunkOne.followedBy(chunkTwo).toList());
-      final Uint8List gzippedChunkOne = gzipped.sublist(0, gzipped.length ~/ 2);
-      final Uint8List gzippedChunkTwo = gzipped.sublist(gzipped.length ~/ 2);
+      final List<int> gzipped = gzip.encode(chunkOne.followedBy(chunkTwo).toList());
+      final List<int> gzippedChunkOne = gzipped.sublist(0, gzipped.length ~/ 2);
+      final List<int> gzippedChunkTwo = gzipped.sublist(gzipped.length ~/ 2);
```

Comment:
I could have sworn there was some change in Dart so this was a `Uint8List` - /cc @tvolkert

Reply:
Some stuff was changed, but encoders/decoders weren't (yet).

Reply:
That was from File/HttpClient APIs, gzip still returns its type as List<int>

Reply:
https://github.com/dart-lang/sdk/issues/36900